code/controllers/subsystem/job.dm

```diff
@@ -543,13 +540,12 @@ SUBSYSTEM_DEF(job)
 
 	if(player_client)
 		if(job.req_admin_notify)
-			to_chat(player_client, "<span class='infoplain'><b>You are playing a job that is important for Game Progression. If you have to disconnect, please notify the admins via adminhelp.</b></span>")
+			to_chat(player_client, "<span class='infoplain'><b>You are playing a job that is important for Game Progression. \
+				If you have to disconnect, please notify the admins via adminhelp.</b></span>")
```

Comment:
```suggestion
			to_chat(player_client, span_infoplain("<b>You are playing a job that is important for Game Progression. \
				If you have to disconnect, please notify the admins via adminhelp.</b>"))
```

Not sure if `<b>` works with with the spanclass helper but I'm going to point it out.